samples/tools/testbed/README.md

```diff
@@ -0,0 +1,134 @@
+# Autogen Testbed Environment
+
+The Autogen Testbed environment is a tool for repeatedly running a set of pre-defined Autogen scenarios in a setting with tightly-controlled initial conditions. With each run, Autogen will start from a blank slate, working out what code needs to be written, and what libraries or dependencies to install. The results of each run are logged, and can be ingested by analysis or metrics scripts. By default, all runs are conducted in freshly-initialized docker containers, providing the recommended level of consistency and safety.
+
+## Setup
+
+Before you begin, you must configure your API keys for use with the Testbed. These keys extend beyond those typically found in an OAI_CONFIG_LIST, and can include such things as keys for the Bing Search API or other services used by the scenarios. There is an example ENV file in ``includes/ENV.example``. To get started:
+
+``cp includes/ENV.example includes/ENV``
+
+Then edit ``includes/ENV`` as needed.
+
+The Testbed also requires installation of the __python docker__ library:
```

Comment:
Clarify the system requirement to run this. Eg I can't run this in code spaces.

Reply:
Good call. I don't use code spaces so I am unfamiliar. It basically requires Docker.

Reply:
Fixed.